### DISCUSSION TITLE
How can I adjust the font size and other CSS of asides in Html output?

### DISCUSSION THREAD

**Discussion by normansimonr** [quarto-dev/quarto-cli#7177] [Q&A] [answered]
### Description

Hi, I have a Quarto website with lots of asides (`[some text]{.aside}`) as margin notes. I want to make the font larger and adjust other CSS parameters, but simply adjusting an `.aside` class in the Styles file doesn't do the trick. I checked the generated Html files and the texts don't appear inside a class called 'aside'. They appear like this:

```html
<span class="">some text</span>
```

Is there a way to adjust the asides' CSS? Thanks!

**Comment by mcanouil** [comment]
See <https://quarto.org/docs/output-formats/html-themes.html> or use your favourite browser developer/inspect mode to target the right elements.

**Reply by normansimonr** [reply]
Hi, thank you very much for your reply. Actually I did that, but I couldn't seem to find the CSS tags in the Bootstrap theming templates. Is there a specific aside variable that I could access otherwise?

**Reply by mcanouil** [reply]
Then you fall in the second part of my answer, *i.e*, browser developer mode to see the CSS/HTML selector you need.

**Reply by mcanouil** [reply, marked as answer]
Now That I have access to a laptop, using developer mode, you can see that the margin elements are in a container with the CSS class: `column-margin`.
````qmd
---
title: "Quarto Playground"
format: html
include-in-header:
  - text: |
      <style>
      .column-margin {
        font-size: 0.5em;
      }
      </style>
---

This is a playground for [Quarto]{.aside}.

::: {.aside}
More aside.
:::
````

You can also tweak each spans/divs:
````qmd
---
title: "Quarto Playground"
format: html
---

This is a playground for [Quarto]{.aside style="font-size:5em;"}.

::: {.aside style="color: red;"}
More aside.
:::
````

**Reply by normansimonr** [reply]
Oh this makes a lot of sense! Thanks so much! I learnt something new today!

**Reply by mcanouil** [reply]
Note that I opened an issue from this discussion to at least ensure there a class in "aside" elements which could help target only those elements in the future (and not other margins elements such as footnotes or references).

- #7183
